cluster/crds/pkg.crossplane.io_functionrevisions.yaml

```diff
@@ -66,8 +66,9 @@ spec:
                   replication controllers and services. More info: http://kubernetes.io/docs/user-guide/labels'
                 type: object
               controllerConfigRef:
-                description: ControllerConfigRef references a ControllerConfig resource
+                description: 'ControllerConfigRef references a ControllerConfig resource
```

Comment:
Looks like it is being added when there are some special characters in the description. In this case, it was added because of `:` after the word `Deprecated`. Given we have other descriptions starting with `'`, that is fine.